ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -100,9 +100,9 @@ void RSkComponentTextInput::drawTextInput(SkCanvas *canvas,
   #if ENABLE(FEATURE_ONSCREEN_KEYBOARD)
   if (0 == displayString_.size()) {
     /*In case of Empty displayString,TextInput renders PlaceHolder Name in TI BOX. To avoid it, passing empty string here */
-    OnScreenKeyboard::updatePlaceHolderString(std::string(),0);
+    OnScreenKeyboard::updatePlaceHolderString(std::string(),0,!caretHidden_);
```

Comment:
is this caretHidden prop required to be supported by OSK.
if yes, then can this be part of OSK Config itself than sending this value during every update of striing

Reply:
Addressed

Reply:
Are we still continuing to pass CaretHidden as Parameter in Update String ? Any Reason for Not setting it through parameter as motioned in the earlier comment

Reply:
Also , Have we confirmed teh requirement to avoid cursor draw for immutable text in OSK in reference platform